adserver/views.py

```diff
@@ -2023,6 +2023,20 @@ def form_valid(self, form):
         )
         return result
 
+    def get(self, request, *args, **kwargs):
+        result = super().get(request, *args, **kwargs)
+
+        if self.object.ignore_mobile_traffic:
+            messages.info(
```

Comment:
Is this how we're displaying other on-page notifications? Seems a bit of an odd use case for notifications vs. an in-page message. This probably works fine though.

Reply:
It's awkward but I didn't want it on every single publisher page.